### PR TITLE
fix broken imports

### PR DESCRIPTION
**Pull request by keskitalo**

### CODE REVIEW

File: sotodlib/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2018-2020 Simons Observatory.
+# Full license can be found in the top level "LICENSE" file.
+"""Simons Observatory utilities.
+
```
Comment (tskisner) [inline sotodlib/utils/__init__.py:4]: Thanks for adding this missing file- slipped through the cracks in the reorganization

File: sotodlib/utils/pipeline_tools/export.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def export_TOD(args, comm, data, totalname, schedules, other=None, verbose=True)
 
     # Only import spt3g if we are writing out so3g files
     from spt3g import core as core3g
-    from ..data.toast_export import ToastExport
+    from sotodlib.io.toast_export import ToastExport
```
Comment (tskisner) [inline sotodlib/utils/pipeline_tools/export.py:52]: We should use relative imports for internal package files.

File: sotodlib/utils/pipeline_tools/hardware.py [hunk 3]
```diff
@@ -7,7 +7,8 @@
 from toast.timing import function_timer, Timer
 from toast.utils import Logger
 
-from .. import hardware
+from sotodlib.core import Hardware
+from sotodlib.sim_hardware import get_example, sim_telescope_detectors
```
Comment (tskisner) [inline sotodlib/utils/pipeline_tools/hardware.py:11]: These should be relative imports.

File: sotodlib/utils/pipeline_tools/noise.py [hunk 4]
```diff
@@ -7,7 +7,8 @@
 from toast.tod import AnalyticNoise
 from toast.utils import Logger
 import toast.qarray as qa
-from .. import hardware
+from sotodlib.sim_hardware import get_example
```
Comment (tskisner) [inline sotodlib/utils/pipeline_tools/noise.py:10]: Should be relative import

File: sotodlib/utils/pipeline_tools/observation.py [hunk 5]
```diff
@@ -184,7 +184,7 @@ def load_observations(args, comm):
     """
     # This import is not at the top of the file to avoid
     # loading spt3g through so3g unnecessarily
-    from ..data.toast_load import load_data
+    from sotodlib.io.toast_load import load_data
```
Comment (tskisner) [inline sotodlib/utils/pipeline_tools/observation.py:187]: Should be relative import

**Comment by keskitalo** [comment]
Should I change these imports as well?
```
sotodlib/g3_condition.py:from sotodlib.core.g3_core import DataG3Module
sotodlib/core/context.py:        from sotodlib.io.load import OBSLOADER_REGISTRY
sotodlib/g3_sim.py:from sotodlib.core.g3_core import DataG3Module
sotodlib/utils/pipeline_tools/hardware.py:                from sotodlib.hardware
sotodlib/g3_filter.py:from sotodlib.core.g3_core import DataG3Module
```

**Comment by tskisner** [comment]
Good catch, yes, thanks!

**Comment by keskitalo** [comment]
I replaced the absolute imports with relative ones.

**Comment by keskitalo** [comment]
@tskisner is this good to merge now?

**Comment by tskisner** [comment]
Yes, I tried to mark my requested changes as "resolved", but should have been more explicit.  Please merge.

**Pull request merged**